recipes/nsight-compute/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "nsight-compute" %}
+{% set version = "2022.4.0.15" %}
+{% set version_split = version.split(".")[0]+"."+version.split(".")[1]+"."+version.split(".")[2] %}
+{% set cuda_version = "12.0" %}
```

Comment:
Tests are failing because this is hardcoded. Should we be skipping when the CUDA version < 12?

Reply:
It looks like tests have since been working. Marking this resolved